com.github.d4nj1.tlpui.yml

```
@@ -0,0 +1,34 @@
+app-id: com.github.d4nj1.tlpui
+command: tlpui
+runtime: org.gnome.Platform
+runtime-version: '3.38'
```

```suggestion
runtime-version: '42'
```

3.38 has been EOL for quite a while.

---

I will test and upgrade to latest version.